rps_game.js

```diff
@@ -64,4 +64,80 @@ document.getElementById('play').addEventListener('click', function () {
   resetComputerDisplay();
   playGame();
 
-});
+}); */
+
+
+(function () {
+  let userInput = {
+    userGesture () {
+      return document.querySelector('input[name=gesture]:checked').value;
+    },
+    clickToPlay () {
+      document.getElementById('play').addEventListener('click', function () {
+        displayReaction.resetComputerDisplay();
+        displayReaction.playGame();
+      });
+    }
+  };
+
+  let computerMove = {
+    _gestureChoice: ['rock', 'paper', 'scissors'],
+
+    get compGesture() {
+      randomIndex = Math.floor(Math.random() * 3);
+      return this._gestureChoice[randomIndex];
+    },
+
+    isDraw(userChoise, computerChoise) {
+      return userChoise === computerChoise;
+    },
+
+    humanWins(userChoise, computerChoise) {
+      return userChoise === 'rock' && computerChoise === 'scissors' ||
+             userChoise === 'paper' && computerChoise === 'rock' ||
+             userChoise === 'scissors' && computerChoise === 'paper';
+    }
+  };
+
+  const displayReaction = {
+    _winSound: new Audio('audio/tada.mp3'),
+    _drawSound: new Audio('audio/kongas.mp3'),
+    _loseSound: new Audio('audio/Wrong-answer-sound-effect.mp3'),
+    _result: document.getElementById('result'),
+
+    resetComputerDisplay() {
+      document.getElementById('c_rock').classList.remove('selected');
+      document.getElementById('c_paper').classList.remove('selected');
+      document.getElementById('c_scissors').classList.remove('selected');
+    },
+
+    _updateDispay(printOnScreen, humanDisplay, robotDisplay) {
+      this._result.textContent = printOnScreen;
+      document.getElementById('human_face').src = humanDisplay;
+      document.getElementById('robot_face').src = robotDisplay;
+    },
+
+    playGame() {
```

Comment:
Good distinction between private (called from inside) and public (called from outside) methods :+1: